php/tests/wrapper_type_setters_test.php

```diff
@@ -225,4 +225,113 @@ public function constructorWithWrapperTypeDataProvider()
             [TestWrapperSetters::class, BytesValue::class, 'bytes_value', 'getBytesValue', "nine"],
         ];
     }
+
+    // /**
+    //  * @dataProvider invalidConstructorWithWrapperTypeDataProvider
+    //  * @expectedException \Exception
+    //  */
+    // public function testInvalidConstructorWithWrapperType($class, $wrapperField, $value)
```

Comment:
I don't want to make it an exception. I want to make it an error.
Our plan is make all type checking an error (which is consistent with native php type hint).
I know previously php implementation throws exception. However, c extension throws error.
(In my mind, this is actually an undefined behavior)

Reply:
Removed